### ISSUE TITLE
0.7.0 fails to build

### ISSUE THREAD

**Issue by compiaffe** [RidgeRun/gst-shark#81]
For background, we are conan packaging gst-shark (and the rest of gstreamer):
On our CICD the latest tag fails to build, the previous 0.6.1 as well as the `test/disable-file-buffering` succeeded.

Errors are:
```
CC       libgstsharktracers_la-gstperiodictracer.lo
In file included from /builds/aivero/public/conan/conan-gstreamer-shark/conan_data/gstreamer/1.16.2/aivero/stable/package/addb6aa6c784b5d625634b9a6a0f65fa3369e937/include/gstreamer-1.0/gst/gst.h:55:0,
                 from gstsharktracer.h:24,
                 from gstperiodictracer.h:24,
                 from gstperiodictracer.c:21:
gstperiodictracer.c: In function 'set_period':
gstperiodictracer.c:248:28: error: format '%s' expects argument of type 'char *', but argument 8 has type 'gpointer {aka void *}' [-Werror=format=]
     GST_INFO_OBJECT (self, "Attempting to parse provided period \"%s\"",
                            ^
         list->data);
         ~~~~~~~             
/builds/aivero/public/conan/conan-gstreamer-shark/conan_data/gstreamer/1.16.2/aivero/stable/package/addb6aa6c784b5d625634b9a6a0f65fa3369e937/include/gstreamer-1.0/gst/gstinfo.h:640:31: note: in definition of macro 'GST_CAT_LEVEL_LOG'
         (GObject *) (object), __VA_ARGS__);    \
                               ^~~~~~~~~~~
gstperiodictracer.c:248:5: note: in expansion of macro 'GST_INFO_OBJECT'
     GST_INFO_OBJECT (self, "Attempting to parse provided period \"%s\"",
     ^~~~~~~~~~~~~~~
gstperiodictracer.c:230:29: error: variable 'priv' set but not used [-Werror=unused-but-set-variable]
   GstPeriodicTracerPrivate *priv = NULL;
                             ^~~~
  CCLD     libgstshark.la
cc1: all warnings being treated as errors
make[2]: *** [Makefile:852: libgstsharktracers_la-gstperiodictracer.lo] Error 1

```
See full output at: https://gitlab.com/aivero/public/conan/conan-gstreamer-shark/-/jobs/630985279

**Comment by michaelgruner** [comment]
Thanks for reporting! I'll be back with a fix in a couple of hours.

**Comment by michaelgruner** [comment]
Fixed on release v0.7.1
